packages/core/src/tests/grpc/config.test.js

```diff
@@ -1,20 +1,26 @@
-let client;
-let coreProcess;
-
+let coreProcess,client,testModule;
 beforeAll(async () => {
   const { exec } = require('child_process');
-  const options = {
+  let options = {
     env: { ...process.env, REDIS_PORT: 6379, REDIS_HOST: 'localhost', PORT: 3030, ADMIN_SOCKET_PORT: 3032 },
```

Comment:
Reorder these so that process env values override the defaults whenever available.